LPS-164650 Add the Expire Action to the Version History for Web Content

jkappler opened the pull request:
## Motivation

[Link to ticket](https://issues.liferay.com/browse/LPS-164650)
[Link to ticket (frontend)](https://issues.liferay.com/browse/LPS-162827)

Tango Team is working on improving the information panel in content dashboard, and one of these improvements is adding the version history of the elements in the sidebar. Each version can have actions available to the user, such as viewing, downloading, sharing, i.e.


## Change summary:

* Replicate the logic that we already have for content dashboard item action (action, provider and tracker), and leave only the barebone
* Change signature so that we pass the version
* Add a new method in content dashboard item to retrieve the actions by version
* Change the resource command to include the actions for each version
* WIP of the frontend to display the actions


## TODO

* Add integration tests

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

jkappler commented:
@cgoncas, not 100% sure about these changes, let me know what you think. I added also the view document so that you can see how it would behave. 

A big thank you to @beltranrengifo, since the code that he made for front worked at once 👏. Also to @Stephy6 for the help with this PR. 

Beltran, I added the last commit without changing anything, so feel free to adapt as you see fit as we talked.

Will wait for your comments Cris

beltranrengifo commented:
This pr resumes the frontend task started [here](https://github.com/liferay-tango/liferay-portal/pull/2885)

cgoncas commented on modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/FileEntryContentDashboardItem.java:
```diff
@@ -215,6 +223,44 @@ public ContentDashboardItemSubtype getContentDashboardItemSubtype() {
 	}
 
+	@Override
+	public List<ContentDashboardItemVersionAction>
```
Hey I think the getContentDashboardItemVersionActions should be added to the ContentDashboardItemVersion, instead the ContentDashboardItem implementation.

beltranrengifo commented:
hey @jkappler @Stephy6 ! as we spoke when I click on an expire option (i.e. 1.0) all versions get expired.

I add a screenshot of the default behaviour in the item history view
![image](https://user-images.githubusercontent.com/19485114/194012618-57392bac-42f3-44b1-a446-ee19e473636a.png)

and other one with the result of clicking in one expire action 
![image](https://user-images.githubusercontent.com/19485114/194013119-b09e52f8-7bf6-46c3-9198-67f99bd28d7e.png)

hope it is helpful!

jkappler commented:
@beltranrengifo, fixed the expired issue :D

jkappler commented on modules/apps/content-dashboard/content-dashboard-api/src/main/java/com/liferay/content/dashboard/item/ContentDashboardItemVersion.java:
```diff
@@ -14,23 +14,36 @@
 public class ContentDashboardItemVersion {
 
 	public ContentDashboardItemVersion(
-		String changeLog, Date createDate, String label, String style,
```
@cgoncas, this would be the new proposal based on the comments that you made

jkappler commented on modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/action/provider/ViewFileVersionContentDashboardItemVersionActionProvider.java:
```diff
@@ -0,0 +1,60 @@
+ */
+@Component(service = ContentDashboardItemVersionActionProvider.class)
+public class ViewFileVersionContentDashboardItemVersionActionProvider
+	implements ContentDashboardItemVersionActionProvider<FileVersion> {
```
@cgoncas, for FileEntries we would register the actions as FileVersions, it will be cleaner at the end

jkappler commented on modules/apps/content-dashboard/content-dashboard-journal-impl/src/main/java/com/liferay/content/dashboard/journal/internal/item/JournalArticleContentDashboardItem.java:
```diff
@@ -460,6 +472,52 @@ public boolean isViewable(HttpServletRequest httpServletRequest) {
 		);
 	}
 
+	private List<ContentDashboardItemVersionAction>
```
@cgoncas, this is the only think I dislike. I think we can extract the code to reuse it, but I'm unsure how....

cgoncas commented:
ci:forward

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/4898/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4898/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-source-format/3419/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/3419/jenkins-report.html">here</a>.</b></p>

jkappler commented:
perfect @cgoncas, I will send the integration tests today. Thanks for reviewing ❤️

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7c33435613651a106b336a4d4bfa3dece7f5f139">7c33435613651a106b336a4d4bfa3dece7f5f139</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/liferay-tango/liferay-portal/tree/LPS-164650_version_actions">LPS-164650_version_actions</a><br/>Branch GIT ID: <a href="https://github.com/liferay-tango/liferay-portal/commit/c03a1ea44cb7caf066a8b8ec665a9dee4d074f44">c03a1ea44cb7caf066a8b8ec665a9dee4d074f44</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/3419/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/3419/jenkins-report.html">here</a>.</h5><pre>     [exec] Generating types (25 of 25): @liferay/object-web
     [exec] 
     [exec] ***************
     [exec] *** WARNING ***
     [exec] ***************
     [exec] 
     [exec] Out-of-date TypeScript build artifacts at:
     [exec] 
     [exec]     apps/object/object-web/types
     [exec] 
     [exec] Please commit them.
     [exec] 
     [exec] M modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/ObjectField/useObjectFieldForm.d.ts
     [exec] 
     [exec] Preflight check failed:
     [exec] checkTypeScriptTypeArtifacts() failed: Error: Type generation was successful but 3 projects were out of date. Please commit updated versions of the artifacts in the projects listed above.
     [exec] 
     [exec] Prettier checked 2 files
     [exec] 
     [exec] 1 of 3 jobs failed
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] error Command failed with exit code 1.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2022-10-06 07:41:26.513.
     [exec] 3 actionable tasks: 2 executed, 1 up-to-date
     [exec] 
     [exec] See the profiling report at: file:///opt/dev/projects/github/liferay-portal/build/reports/profile/profile-2022-10-06-00-39-13.html
     [exec] A fine-grained performance profile is available: use the --scan option.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$3(ExecuteActionsTaskExecuter.java:186)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:268)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:184)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:173)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/3419/">test-portal-source-format#3419</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/3419/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-tango/liferay-portal/pull/2900">liferay-tango#2900</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1998841057">[master] ci:test:sf - liferay-tango &gt; liferay-tango - PR#2900 - 2022-10-06[00:36:25]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/12425/">publish-testray-report#12425</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7c33435613651a106b336a4d4bfa3dece7f5f139">7c33435613651a106b336a4d4bfa3dece7f5f139</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/liferay-tango/liferay-portal/tree/LPS-164650_version_actions">LPS-164650_version_actions</a><br/>Branch GIT ID: <a href="https://github.com/liferay-tango/liferay-portal/commit/c03a1ea44cb7caf066a8b8ec665a9dee4d074f44">c03a1ea44cb7caf066a8b8ec665a9dee4d074f44</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-source-format/3515/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/3515/jenkins-report.html">here</a>.</h5><pre>     [exec] Generating types (21 of 25): @liferay/notification-web
     [exec] Generating types (22 of 25): @liferay/oauth2-provider-web
     [exec] Generating types (23 of 25): @liferay/object-dynamic-data-mapping-form-field-type
     [exec] Generating types (24 of 25): @liferay/object-js-components-web
     [exec] Generating types (25 of 25): @liferay/object-web
     [exec] 
     [exec] ***************
     [exec] *** WARNING ***
     [exec] ***************
     [exec] 
     [exec] Out-of-date TypeScript build artifacts at:
     [exec] 
     [exec]     apps/object/object-web/types
     [exec] 
     [exec] Please commit them.
     [exec] 
     [exec] M modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/ObjectField/useObjectFieldForm.d.ts
     [exec] 
     [exec] Preflight check failed:
     [exec] checkTypeScriptTypeArtifacts() failed: Error: Type generation was successful but 3 projects were out of date. Please commit updated versions of the artifacts in the projects listed above.
     [exec] 
     [exec] Prettier checked 2 files
     [exec] 
     [exec] 1 of 3 jobs failed
     [exec] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
     [exec] error Command failed with exit code 1.
     [exec] 
     [exec] &gt; Task :packageRunCheckFormat FAILED
     [exec] Gradle build finished at 2022-10-06 07:51:57.069.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$3(ExecuteActionsTaskExecuter.java:186)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:268)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:184)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:173)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/3515/">test-portal-source-format#3515</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/3515/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-tango/liferay-portal/pull/2900">liferay-tango#2900</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1999658658">[master] ci:test:sf - liferay-tango &gt; liferay-tango - PR#2900 - 2022-10-06[00:46:48]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/12910/">publish-testray-report#12910</a><br/></div></div>

beltranrengifo reviewed (approved):
working like a charm @jkappler @Stephy6 🎉

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 28 out of 28 jobs passed</h3><h3>:x: ci:test:relevant - 48 out of 50 jobs passed in 1 hour 42 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/3872">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1307401_4898</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7c33435613651a106b336a4d4bfa3dece7f5f139">7c33435613651a106b336a4d4bfa3dece7f5f139</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6ccd49b01d62b6d4d528d869a544e7c92db97f73">6ccd49b01d62b6d4d528d869a544e7c92db97f73</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1998267489">EE Development Acceptance (master) - 3871 - 2022-10-05[07:35:59]</a></p><details><summary><strong>ci:test:stable - 28 out of 28 jobs PASSED</strong></summary><details><summary><strong>28 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 48 out of 50 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/4898/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268962/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224402/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>empty-temp-dir-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204553/">empty-temp-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259191/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228204/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277277/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261618/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277280/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237849/"><strike>functional-upgrade-tomcat90-mysql57-jdk8/0/1</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264467/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259192/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46679/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261616/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251561/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46676/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277276/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46677/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251559/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237847/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224401/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228203/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277278/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237848/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251560/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248197/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204550/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261620/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46678/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204552/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237850/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259193/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264468/">modules-integration-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228205/">modules-integration-mysql57-jdk8/0/3</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248199/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224403/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277279/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248198/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46680/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251562/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261619/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264465/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237846/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/229110/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261617/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261621/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268963/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264466/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204551/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/229109/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>48 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4898/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237849//consoleText">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a><pre>     [exec] * What went wrong:
     [exec] Execution failed for task ':runPoshi'.
     [exec] &gt; There were failing tests. See the report at: file:///opt/dev/projects/github/liferay-portal/portal-web/tests/index.html
     [exec] 
     [exec] * Try:
     [exec] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Get more help at https://help.gradle.org
     [exec] 
     [exec] BUILD FAILED in 6m 0s
     [exec] Result: 1
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/237849
     [copy] Copying 3 files to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/237849/screenshots
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

clean-up-selenium-driver:
     [echo] Executing commands: 
     [echo] killall chromedriver
     [exec] chromedriver: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

clean-up-logs:
     [echo] stop.app.server.0.start.timestamp: 10-06-2022 02:17:16:956 PDT
[beanshell] 2022-10-06 09:17:27.374 INFO  [main][ThemeHotDeployListener:171] 1 theme for minium-theme was unregistered
[beanshell] 2022-10-06 09:17:27.410 INFO  [main][ThemeHotDeployListener:171] 1 theme for admin-theme was unregistered
[beanshell] 2022-10-06 09:17:27.780 INFO  [main][ThemeHotDeployListener:171] 1 theme for dialect-theme was unregistered
[beanshell] 2022-10-06 09:17:28.050 INFO  [main][ThemeHotDeployListener:171] 1 theme for speedwell-theme was unregistered
[beanshell] 2022-10-06 09:17:28.171 INFO  [main][ThemeHotDeployListener:171] 1 theme for classic-theme was unregistered
[stopwatch] [stop.app.server.0: 16.523 sec]
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

run-poshi-evaluate-console:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null887023235.properties

prepare-poshi-runner-properties:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/portal-web/poshi-ext.properties
[propertyfile] Updating property file: /opt/dev/projects/github/liferay-portal/.gradle/gradle.properties</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

cgoncas commented:
ci:reevaluate:1307401_4898

liferay-continuous-integration commented:
CI is reevaluating the build with build ID: `1307401_4898` against the latest valid upstream results.

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/4898/">test-portal-acceptance-pullrequest(master)#4898</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4898/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-tango/liferay-portal/pull/2900">liferay-tango#2900</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1999700083">[master] ci:test:relevant - liferay-tango &gt; liferay-tango - PR#2900 - 2022-10-06[00:36:24]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/12775/">publish-testray-report#12775</a><br/></div></div>

liferay-continuous-integration commented:
<html><p>Previous <code>ci:test:relevant</code><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/4898">build</a> was unsuccessfully <a href="https://test-1-35.liferay.com/job/test-portal-evaluate-pullrequest/75/">reevaluated</a> against more recent upstream results.</p><h3>:heavy_check_mark: ci:test:stable - 26 out of 26 jobs passed</h3><h3>:x: ci:test:relevant - 48 out of 50 jobs passed in 1 hour 44 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/3872">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1307401_4898</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7c33435613651a106b336a4d4bfa3dece7f5f139">7c33435613651a106b336a4d4bfa3dece7f5f139</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6ccd49b01d62b6d4d528d869a544e7c92db97f73">6ccd49b01d62b6d4d528d869a544e7c92db97f73</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1998267489">EE Development Acceptance (master) - 3871 - 2022-10-05[07:35:59]</a></p><details><summary><strong>ci:test:stable - 26 out of 26 jobs PASSED</strong></summary><details><summary><strong>26 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 48 out of 50 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/4898/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268962/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224402/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>empty-temp-dir-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204553/">empty-temp-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259191/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228204/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277277/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261618/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277280/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237849/"><strike>functional-upgrade-tomcat90-mysql57-jdk8/0/1</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264467/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259192/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46679/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261616/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251561/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46676/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277276/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46677/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251559/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237847/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224401/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228203/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277278/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237848/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251560/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248197/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204550/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261620/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46678/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204552/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237850/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259193/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264468/">modules-integration-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228205/">modules-integration-mysql57-jdk8/0/3</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248199/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224403/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277279/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248198/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46680/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251562/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261619/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264465/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237846/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/229110/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261617/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261621/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268963/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264466/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204551/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/229109/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>48 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4898/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237849//consoleText">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a><pre>     [exec] * What went wrong:
     [exec] Execution failed for task ':runPoshi'.
     [exec] &gt; There were failing tests. See the report at: file:///opt/dev/projects/github/liferay-portal/portal-web/tests/index.html
     [exec] 
     [exec] * Try:
     [exec] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Get more help at https://help.gradle.org
     [exec] 
     [exec] BUILD FAILED in 6m 0s
     [exec] Result: 1
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/237849
     [copy] Copying 3 files to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/237849/screenshots
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

clean-up-selenium-driver:
     [echo] Executing commands: 
     [echo] killall chromedriver
     [exec] chromedriver: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

clean-up-logs:
     [echo] stop.app.server.0.start.timestamp: 10-06-2022 02:17:16:956 PDT
[beanshell] 2022-10-06 09:17:27.374 INFO  [main][ThemeHotDeployListener:171] 1 theme for minium-theme was unregistered
[beanshell] 2022-10-06 09:17:27.410 INFO  [main][ThemeHotDeployListener:171] 1 theme for admin-theme was unregistered
[beanshell] 2022-10-06 09:17:27.780 INFO  [main][ThemeHotDeployListener:171] 1 theme for dialect-theme was unregistered
[beanshell] 2022-10-06 09:17:28.050 INFO  [main][ThemeHotDeployListener:171] 1 theme for speedwell-theme was unregistered
[beanshell] 2022-10-06 09:17:28.171 INFO  [main][ThemeHotDeployListener:171] 1 theme for classic-theme was unregistered
[stopwatch] [stop.app.server.0: 16.523 sec]
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

run-poshi-evaluate-console:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null887023235.properties

prepare-poshi-runner-properties:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/portal-web/poshi-ext.properties
[propertyfile] Updating property file: /opt/dev/projects/github/liferay-portal/.gradle/gradle.properties</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

cgoncas commented:
ci:reevaluate:1307401_4898

liferay-continuous-integration commented:
CI is reevaluating the build with build ID: `1307401_4898` against the latest valid upstream results.

cgoncas commented:
Hey @CarlosBrichete and @Luizcarlosqueiroz !!

Could you take a look to this PR, so if not, we can send it directly to Brian.

Thanks!!

liferay-continuous-integration commented:
<html><p>Previous <code>ci:test:relevant</code><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/4898">build</a> was unsuccessfully <a href="https://test-1-37.liferay.com/job/test-portal-evaluate-pullrequest/167/">reevaluated</a> against more recent upstream results.</p><h3>:heavy_check_mark: ci:test:stable - 26 out of 26 jobs passed</h3><h3>:x: ci:test:relevant - 48 out of 50 jobs passed in 1 hour 44 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/3872">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1307401_4898</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7c33435613651a106b336a4d4bfa3dece7f5f139">7c33435613651a106b336a4d4bfa3dece7f5f139</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/6ccd49b01d62b6d4d528d869a544e7c92db97f73">6ccd49b01d62b6d4d528d869a544e7c92db97f73</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1998267489">EE Development Acceptance (master) - 3871 - 2022-10-05[07:35:59]</a></p><details><summary><strong>ci:test:stable - 26 out of 26 jobs PASSED</strong></summary><details><summary><strong>26 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 48 out of 50 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/4898/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268962/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224402/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>empty-temp-dir-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204553/">empty-temp-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259191/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228204/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277277/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261618/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277280/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237849/"><strike>functional-upgrade-tomcat90-mysql57-jdk8/0/1</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264467/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259192/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46679/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261616/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251561/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46676/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277276/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46677/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251559/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237847/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224401/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228203/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277278/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237848/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251560/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248197/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204550/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261620/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46678/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204552/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237850/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/259193/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264468/">modules-integration-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228205/">modules-integration-mysql57-jdk8/0/3</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248199/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224403/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/277279/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/248198/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/46680/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251562/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261619/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264465/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237846/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/229110/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261617/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/261621/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268963/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/264466/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/204551/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/229109/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>48 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4898/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/237849//consoleText">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a><pre>     [exec] * What went wrong:
     [exec] Execution failed for task ':runPoshi'.
     [exec] &gt; There were failing tests. See the report at: file:///opt/dev/projects/github/liferay-portal/portal-web/tests/index.html
     [exec] 
     [exec] * Try:
     [exec] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Get more help at https://help.gradle.org
     [exec] 
     [exec] BUILD FAILED in 6m 0s
     [exec] Result: 1
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/237849
     [copy] Copying 3 files to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/237849/screenshots
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

clean-up-selenium-driver:
     [echo] Executing commands: 
     [echo] killall chromedriver
     [exec] chromedriver: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

clean-up-logs:
     [echo] stop.app.server.0.start.timestamp: 10-06-2022 02:17:16:956 PDT
[beanshell] 2022-10-06 09:17:27.374 INFO  [main][ThemeHotDeployListener:171] 1 theme for minium-theme was unregistered
[beanshell] 2022-10-06 09:17:27.410 INFO  [main][ThemeHotDeployListener:171] 1 theme for admin-theme was unregistered
[beanshell] 2022-10-06 09:17:27.780 INFO  [main][ThemeHotDeployListener:171] 1 theme for dialect-theme was unregistered
[beanshell] 2022-10-06 09:17:28.050 INFO  [main][ThemeHotDeployListener:171] 1 theme for speedwell-theme was unregistered
[beanshell] 2022-10-06 09:17:28.171 INFO  [main][ThemeHotDeployListener:171] 1 theme for classic-theme was unregistered
[stopwatch] [stop.app.server.0: 16.523 sec]
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null514454027.properties

run-poshi-evaluate-console:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null887023235.properties

prepare-poshi-runner-properties:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/portal-web/poshi-ext.properties
[propertyfile] Updating property file: /opt/dev/projects/github/liferay-portal/.gradle/gradle.properties</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

jkappler commented:
Sent directly to BChan here: https://github.com/brianchandotcom/liferay-portal/pull/124182
